src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -63,6 +65,49 @@ Amplify.configure({
 });
 ```
 
+### Lex V2 bot
+You can create Amazon Lex V2 chatbox in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V2 Developer Guide](https://docs.aws.amazon.com/lexv2/latest/dg/getting-started.html).
+
+![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
+
+With manual setup, you need to provide your auth credentials and bot details to configure your app:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+import { AWSLexV2Provider } from '@aws-amplify/interactions';
+
+Amplify.addPluggable(new AWSLexV2Provider());
+
+const interactionsConfig = {
+    Auth: {
+        identityPoolId: "<identityPoolId>",
+        region: "<region>"
+    },
+    Interactions: {
+        bots: {
+            // LexV1 Bot
+            <V1BotName>: {
+                name: "<V1BotName>",
+                alias: "<V1BotAliasName>",
+                region: "<V1BotRegion>",
+                // default provider "AWSLexProvider"
+            },
```

Comment:
I would remove this and focus on Lex V2 on this section

Reply:
makes sense, updated